triggers_windows.go

```diff
@@ -0,0 +1,65 @@
+package jasper
+
+import (
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+const (
+	// The name for the mongod termination event has the format "Global\Mongo_<pid>".
+	mongodShutdownEventNamePrefix = "Global\\Mongo_"
+	mongodShutdownEventTimeout    = 60 * time.Second
+)
+
+// makeMongodEventTrigger is necessary to support clean termination of mongods on Windows by signaling the mongod
+// shutdown event and waiting for the process to return.
+func makeMongodShutdownSignalTrigger() SignalTrigger {
+	return func(info ProcessInfo, sig syscall.Signal) (skipSignal bool) {
```

Comment:
I would not do the declaration here.

Reply:
Where would you prefer to put this declaration? As in it should go in a different file?

Reply:
sorry, I would avoid declaring the skipSignal variable in the return statement, and just return false or true in each statement.